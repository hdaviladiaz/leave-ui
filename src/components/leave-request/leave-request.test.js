'use strict';
import React from 'react';
import { shallow } from 'enzyme';
import LeaveRequest from './leave-request.js';

describe('LeaveRequest', () => {

  it('should render LeaveRequest component', () => {
    const wrapper = shallow(<LeaveRequest />);
    const instanceLeaveRequest = wrapper.instance();
    expect(instanceLeaveRequest).toBeInstanceOf(LeaveRequest);
  });
});
