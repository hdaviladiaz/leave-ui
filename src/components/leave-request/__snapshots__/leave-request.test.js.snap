// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LeaveRequest as snapshot 1`] = `
<div
  className="leave-request text-center panel panel-default"
  id={undefined}
>
  <div
    className="panel-body"
  >
    <img
      alt=""
      className="back-image"
      src="/img/back.png"
    />
    <h4
      className="gray-color"
    >
      No has solicitado vacaciones
    </h4>
  </div>
</div>
`;
